html/elements/script.json

```
@@ -558,13 +558,16 @@
             "__compat": {
               "support": {
                 "chrome": {
-                  "version_added": "61"
+                  "version_added": "61",
+                  "notes": "Chrome does not load module scripts without the <code>async</code> attribute when the page is served as XHTML (<code>application/xhtml+xml</code>) - <a href='https://crbug.com/717643'>Chromium Issue #717643</a>"
                 },
                 "chrome_android": {
-                  "version_added": "61"
+                  "version_added": "61",
+                  "notes": "Chrome does not load module scripts without the <code>async</code> attribute when the page is served as XHTML (<code>application/xhtml+xml</code>) - <a href='https://crbug.com/717643'>Chromium Issue #717643</a>"
                 },
                 "edge": {
-                  "version_added": "16"
+                  "version_added": "16",
+                  "notes": "Chrome does not load module scripts without the <code>async</code> attribute when the page is served as XHTML (<code>application/xhtml+xml</code>) - <a href='https://crbug.com/717643'>Chromium Issue #717643</a>"
```

We also try to avoid mentioning unrelated browsers in notes. You can either rework the note to avoid the browser name (when it makes sense to do so), like this:

```suggestion
                  "notes": "Module scripts without the <code>async</code> attribute do not load when the page is served as XHTML (<code>application/xhtml+xml</code>). See <a href='https://crbug.com/717643'>bug 717643</a>."
```

(this is nice, as you can copy and paste to several notes)

Or you can substitute the correct browser name, like this:


```suggestion
                  "notes": "Edge does not load module scripts without the <code>async</code> attribute when the page is served as XHTML (<code>application/xhtml+xml</code>). See <a href='https://crbug.com/717643'>bug 717643</a>."
```

---

I'm not sure about how anyone else would like it formatted, but I feel like saying "Chromium" (I'm not sure why I used Chrome everywhere though) would more accurately reflect the underlying reason behind these particular browsers not supporting this feature.

It's not that a particular browser didn't implement it, instead, it's because they all use the same software as a frame, which doesn't support it.

But, if you disagree, I could commit one of your suggestions instead.

---

Early in this project's history, there was was decision made to avoid engine names where possible and use the names of browsers. Apparently, these names are more widely recognized among developers than the engines. Thanks for your understanding on this.